### PR TITLE
OPSEXP-1216 release acs 7.1.1

### PR DESCRIPTION
**Pull request by gionn**
ref OPSEXP-1216

### CODE REVIEW

**Comment by gionn** [comment]
```
TASK [adw : Check if checksum is available for version 2.6.0] **********************************************************
[WARNING]: An unhandled exception occurred while running the lookup plugin 'url'. Error was a <class
'ansible.errors.AnsibleError'>, original message: Received HTTP error for
https://artifacts.alfresco.com/nexus/content/groups/private/org/alfresco/alfresco-digital-workspace/2.6.0/alfresco-
digital-workspace-2.6.0.zip.sha1 : HTTP Error 404: Not Found
ok: [instance]

TASK [adw : Download alfresco-digital-workspace-2.6.0.zip] *************************************************************
ok: [instance]
```

File: molecule/default/verify.yml [hunk 1]
```diff
@@ -30,12 +30,20 @@
...
-          selfsigned_cert_path: "{{ project_dir }}/configuration_files/ssl_certificates/{{ ansible_ec2_public_hostname }}.crt"
-      ansible.builtin.shell:
-        chdir: "{{ dta_dir }}"
-        cmd: |
-          export REQUESTS_CA_BUNDLE={{ selfsigned_cert_path }}
-          pytest --tb=line --configuration {{ test_config_file }} tests/ -s
+    - name: Run tests
+      block:
+        - name: Run pytest
+          delegate_to: localhost
```
Comment (alxgomz) [inline molecule/default/verify.yml:36]: Can't we run this playbook on localhost only?

**Review by alxgomz — CHANGES_REQUESTED** [review]
Also the VERSION file needs to be bumped to 1.2

File: molecule/default/verify.yml (hunk 1 above)
Comment (gionn) [reply molecule/default/verify.yml:36]: we are addressing this in the multimachine PR

Comment (alxgomz) [reply molecule/default/verify.yml:36]: Ah ok sorry I'm getting confused now :)

**Review by alxgomz — APPROVED** [review]
LGTM

**Pull request merged**